cpp/src/io/json/json_normalization.cu

```diff
@@ -298,10 +299,11 @@ struct TransduceToNormalizedWS {
 
 namespace detail {
 
-void normalize_single_quotes(datasource::owning_buffer<rmm::device_uvector<SymbolT>>& indata,
+void normalize_single_quotes(datasource::owning_buffer<rmm::device_buffer>& indata,
                              rmm::cuda_stream_view stream,
                              rmm::device_async_resource_ref mr)
 {
+  CUDF_FUNC_RANGE();
```

Comment:
```suggestion
```
These are not generally included in detail functions. There is no public API function for this API?
https://github.com/rapidsai/cudf/blob/branch-24.10/cpp/doxygen/developer_guide/DEVELOPER_GUIDE.md#external